packages/nbdime/src/styles/variables.css

```diff
@@ -2,17 +2,17 @@
 :root {
   --codemirror-border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
 
-  --jp-nbdime-output-color1: rgba(0, 141, 255, 0.7);
+  --jp-nbdime-output-color1: rgba(225, 224, 178, 1.0);
   --jp-nbdime-output-color2: rgba(0, 141, 255, 0.5);
-  --jp-nbdime-output-color3: rgba(0, 141, 255, 0.3);
+  --jp-nbdime-output-color3: rgba(225, 224, 178, 0.25);
 
   --jp-diff-added-color0: rgba(0, 200, 20, 0.6);
   --jp-diff-added-color1: rgba(0, 200, 20, 0.3);
-  --jp-diff-added-color2: rgba(0, 200, 20, 0.3);
+  --jp-diff-added-color2: rgba(225, 244, 221, 0.3);
 
   --jp-diff-deleted-color0: rgba(240, 20, 0, 0.55);
   --jp-diff-deleted-color1: rgba(255, 20, 0, 0.3);
-  --jp-diff-deleted-color2: rgba(255, 20, 0, 0.3);
+  --jp-diff-deleted-color2: rgba(244, 221, 221, 1.0);
```

Comment:
Actually, I think this is the only color that looks bad in dark theme because of the increased alpha.

Reply:
I can come up with another set of colors for dark UI

Reply:
This is technically possible due to lab tagging the DOM with dark/light info, but note that this should work for a wide range of syntax highlighting colors as well.

Reply:
Also, the dark/light tag is more of a last resort kind of mechanism. So if you find a color specification that works on both dark and light, that would be best I think.

Reply:
![image](https://user-images.githubusercontent.com/551333/59543222-bbf77d00-8ebe-11e9-9553-af35f1e9efb8.png)
Revised color palette to work for both dark/light

Reply:
🔥

Reply:
Addressed in latest commit